Model-ID-Reference.md

```diff
@@ -2454,13 +2454,61 @@
 | 2468         | Altana???                                         |
 | 2500         |                                                   |
 | 2678         | Arciela                                           |
+| 2832         | Voodoo Doll                                       |
+| 2850         | Loupan (Fire)                                     |
```

Comment:
Luopan *

Reply:
fixed here, another pr for our src coming up